python/lsst/display/matplotlib/matplotlib.py

```diff
@@ -696,11 +850,13 @@ def __init__(self, image=None, Q=8):
 
         See Lupton et al., PASP 116, 133
         """
-        Normalize.__init__(self)
 
         # The object used to perform the desired mapping
         self.mapping = afwRgb.AsinhZScaleMapping(image, Q)
 
+        vmin, vmax = self._getMinMaxQ()[0:2]
+        Normalize.__init__(self, vmin, vmax)
```

Comment:
Add a comment if you are deliberately bypassing the `AsinhNormalize.__init__` to explain why `super()` is not being used.